sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/epkversion/LeaseStoreManagerImpl.java

```diff
@@ -205,6 +206,11 @@ public Mono<Void> delete(Lease lease) {
             .then();
     }
 
+    @Override
+    public Mono<Void> delete(List<Lease> leases) {
+        throw new UnsupportedOperationException("Delete with leases are not supported for Change Feed epk version wire format");
```

Comment:
On first glance I would ask why you even added this to the base class? Handling UnsupportedException to recognize seems worse than letting compiler to pick it up? But maybe further review will help to answer that question...